onnxruntime/python/tools/tensorrt/perf/build/checkout_trt.sh

```diff
@@ -8,11 +8,11 @@ then
     git remote update
     if [ "$1" = "8.0" ]
     then 
-        git checkout "$1"'-GA'
+        git checkout 1f416bb
```

Comment:
why this specific commit?

Reply:
8.0-GA causes a build failure, Chi told me this to run with this commit, which made ORT able to build.

Reply:
that looks odd. 8.0-GA worked fine when we were developing TRT EP support for 8.0. is it an issue with the latest onnxruntime master? does the rel-1.10.0 branch still work? @chilo-ms can you provide additional context about this?

Reply:
I'm using the branch [trt8debug](https://github.com/microsoft/onnxruntime/tree/trt8debug/cmake/external) to compare and debug regression on trt8.2. The onnx-tensorrt uses that commit.

Reply:
Confirmed, 8.0-GA does work, sending an email with more information regarding master shortly.